src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
```

Comment:
```js
import React from "react";
```

Not required from React 17 because of new JSX Transform. [Read More](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)

Remove this line.